modules/backup-gcs/client.go

```diff
@@ -98,42 +98,52 @@ func (g *gcsClient) getObject(ctx context.Context, bucket *storage.BucketHandle,
 	return content, nil
 }
 
-func (g *gcsClient) HomeDir(backupID string) string {
+func (g *gcsClient) HomeDir(backupID, overrideBucket, overridePath string) string {
 	return "gs://" + path.Join(g.config.Bucket,
-		g.makeObjectName(backupID))
+		g.makeObjectName(overridePath, []string{backupID}))
 }
 
-func (g *gcsClient) findBucket(ctx context.Context) (*storage.BucketHandle, error) {
-	bucket := g.client.Bucket(g.config.Bucket)
+func (g *gcsClient) findBucket(ctx context.Context, bucketOverride string) (*storage.BucketHandle, error) {
+	b := g.config.Bucket
+
+	if bucketOverride != "" {
+		b = bucketOverride
+	}
+	bucket := g.client.Bucket(b)
 
 	if _, err := bucket.Attrs(ctx); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("find bucket: '%w'", err)
```

Comment:
please remove single quotes `'%w'` when we wrap the errors

Reply:
done